components/Starknet/modules/tools/pages/bridged_tokens.adoc

```diff
@@ -1,20 +1,33 @@
 [id="bridged_tokens"]
 = Bridged tokens
 
-The tokens that are currently bridged to Starknet are listed in the following `.json` files:
-
-[horizontal,labelwidth="20"]
-link:https://github.com/starknet-io/starknet-addresses/blob/master/bridged_tokens/mainnet.json[`mainnet.json`^]:: The addresses of the tokens currently bridged to Starknet Mainnet.
-link:https://github.com/starknet-io/starknet-addresses/blob/master/bridged_tokens/goerli.json[`goerli.json`^]:: The addresses of the tokens currently bridged to Starknet Goerli testnet.
-link:https://github.com/starknet-io/starknet-addresses/blob/master/bridged_tokens/sepolia.json[`sepolia.json`^]:: The addresses of the tokens currently bridged to Starknet Sepolia testnet.
-
-Each token has the following parameters:
-
-[horizontal, labelwidth="20"]
-`name`:: Token name.
-`symbol`:: Token symbol.
-`decimals`:: Number of decimal places used to get the user representation.
-`l1_token_address`:: Address of the L1 ERC-20 contract.
-`l2_token_address`:: Address of the L2 ERC-20 contract.
-`l1_bridge_address`:: Address of the L1 bridge contract.
-`l2_bridge_address`:: Address of the L2 bridge contract.
+*The tokens that are currently bridged to Starknet are listed in the following `.json` files:*
```

Comment:
Avoid formatting sentences with bold unless it's really necessary. We want to preserve markup language semantics, and using hard-coded formatting interferes with the page semantics.

If you need to make a sentence bold as a caption or title without adding structure via an actual heading, AsciiDoc provides dot headings:

`.A dot followed by text will be bold`

I use this syntax for the word *Procedure* to visually set off the beginning of a procedure.

In this case, I suggest keeping this sentence non-bold, and adding a title to the table by putting a doc heading with the title text on the line immediately preceding the beginning of the table.